integration-tests/http/__tests__/product/admin/product.spec.ts

```diff
@@ -1262,14 +1262,16 @@ medusaIntegrationTestRunner({
           expect(response.data.product).toEqual(
             expect.objectContaining({
               title: "Test product - 1",
-              variants: [
+              variants: expect.arrayContaining([
```

Comment:
If we use `arrayContaining` then the result can come in either order, and we want to check here that they do come in the specified order right?